cpp/oneapi/dal/algo/jaccard/detail/vertex_similarity_ops.hpp

```diff
@@ -58,7 +56,11 @@ struct vertex_similarity_ops {
         if (column_begin > column_end) {
             throw invalid_argument(msg::column_begin_gt_column_end());
         }
-        if (row_end > vertex_count || column_end > vertex_count) {
+        const auto vertex_count =
+            (oneapi::dal::preview::detail::get_impl(input.get_graph())->_vertex_count);
+        // Safe conversion as ranges were checked
+        if (static_cast<std::size_t>(row_end) > vertex_count ||
+            static_cast<std::size_t>(column_end) > vertex_count) {
```

Comment:
double check: row_end and column_end are exclusive indexes? I mean if I need only 0,1,2,3 columns, should I set column_end =4?

Reply:
Yes, it is [) like interval.